### PR TITLE
feat(suite-native): make coin enabling list items pressable

### PR DESCRIPTION
**Pull request by vytick**
let users enable/disable networks via tapping on list item not only switch. Requested by @shenkys

### CODE REVIEW

File: suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx [hunk 1]
```diff
@@ -96,14 +96,16 @@ export const NetworkSymbolSwitchItem = ({
...
-        <Card style={applyStyle(wrapperStyle, { isEnabled })}>
-            <View style={applyStyle(iconWrapperStyle)}>
-                <CryptoIcon symbol={networkSymbol} />
-            </View>
-            <HStack justifyContent="space-between" spacing={12} flex={1} alignItems="center">
-                <Text variant="callout">{name}</Text>
-                <Switch onChange={handleEnabledChange} isChecked={isEnabled} />
-            </HStack>
-        </Card>
+        <Pressable onPress={_ => handleEnabledChange(!isEnabled)} accessibilityRole="togglebutton">
```
Comment (Nodonisko) [inline suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx]: Why not touchable opacity so you have some feedback actually?

Comment (vytick) [reply suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx]: 0b6473418b

File: suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx [hunk 2]
```diff
@@ -96,7 +96,11 @@ export const NetworkSymbolSwitchItem = ({
     };
 
     return (
-        <Pressable onPress={_ => handleEnabledChange(!isEnabled)} accessibilityRole="togglebutton">
+        <TouchableOpacity
+            onPress={_ => handleEnabledChange(!isEnabled)}
+            accessibilityRole="togglebutton"
+            activeOpacity={0.6}
```
Comment (Nodonisko) [inline suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx:102]: Why overriding default opacity value?

Comment (vytick) [reply suite-native/coin-enabling/src/components/NetworkSymbolSwitchItem.tsx:102]: @shenkys mentioned before (in add new coin), that default 0.2 is too much transparent.

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**